Source/WebKit/Scripts/webkit/parser.py

```diff
@@ -69,16 +69,22 @@ def parse(file):
             elif line.startswith('#else') or line.startswith('#elif'):
                 raise Exception("ERROR: '%s' is not supported in the *.in files" % line)
             continue
-        match = re.search(r'([A-Za-z_0-9]+)\((.*?)\)(?:(?:\s+->\s+)\((.*?)\))?(?:\s+(.*))?', line)
+        match = re.search(r'(?:\[(.*)\] (?:.* )?)?([A-Za-z_0-9]+)\((.*?)\)(?:(?:\s+->\s+)\((.*?)\))?(?:\s+(.*))?', line)
         if match:
-            name, parameters_string, reply_parameters_string, attributes_string = match.groups()
+            options_string, name, parameters_string, reply_parameters_string, attributes_string = match.groups()
             if parameters_string:
                 parameters = parse_parameters_string(parameters_string)
                 for parameter in parameters:
                     parameter.condition = combine_condition(conditions)
             else:
                 parameters = []
 
+            runtime_enablement = None
+            if options_string:
+                match = re.search(r"(?:(?:, |^)+(?:EnabledIf='(.*)'))(?:, |$)?", options_string)
```

Comment:
Add something in Source/WebKit/Scripts/webkit/test that covers this and run make in the same directory to update the test expectations.